src/gpu/helpersKernels.cu

```diff
@@ -30,4 +30,18 @@ void gpuSetupRandomNumberGenerator(cudaStream_t stream, size_t elementsCount, un
                                    curandStatePhilox4_32_10_t* outPHILOXStates)
 {
 	run(kSetupRandomNumberGenerator, stream, elementsCount, seed, outPHILOXStates);
-}
\ No newline at end of file
+}
+
+__global__ void kUpdateVertices(size_t vertexCount, Vec3f* newVerticesToVelocity, Vec3f* oldToNewVertices)
```

Comment:
```suggestion
// Updates vertices and calculates their displacement.
// Input: newVertices and oldVertices
// Output: verticesDisplacement and newVertices
__global__ void kUpdateVertices(size_t vertexCount, Vec3f* newVerticesToDisplacement, Vec3f* oldToNewVertices)
```
1. Such a comment clarifies what is going on here (no need to refer to Mesh::updateVertices and wasting time)
2. Renaming `newVerticesToVelocity` to `newVerticesToDisplacement`